src/main.rs

```diff
@@ -52,123 +53,138 @@ async fn start_websocket(handler: Arc<Mutex<Handler>>) {
 
     warp::serve(axolotl_route).run(([127, 0, 0, 1], 9080)).await;
     log::info!("Server stopped");
-    // Why do we want to lock it shortly on shutdown?
-    let _ = handler_mutex.lock().await;
```

Comment:
I hope there was no secret reason behind this line...

Reply:
lets try :)